src/SearchInput/SearchInput.tsx

```diff
@@ -58,18 +66,31 @@ export const SearchInput = forwardRef<HTMLInputElement, SearchInputProps>(
       initialState: null,
       stateProp: value,
     })
-
     const [searchQuery, setSearchQuery] = useState<string | null>(null)
 
+    const fuse = useMemo(() => {
+      const searchKeys = ['label', 'value']
+      return new Fuse(searchList, {
+        shouldSort: true,
+        keys: searchKeys,
+        findAllMatches: true,
+        minMatchCharLength: 2,
```

Comment:
I've set some defaults here. as we only have a `label` and `value` on the options, these are the only keys we can use so these can remain static. The rest can potentially refined at a later date if we need to but in tl;dr:

**minMatchCharLength**: Requires at least 2 characters to find a match
**should sort**: auto sorts results based on the "weight" given to them
**findAllMatches**: finds all matches even if an exact match is found e.g. searching for if a list contains both `Analyst` and `Data Analyst` and the user searches for `Analyst`, both will be returned but `Analyst` will be top as it is an exact match